website/docs/r/cluster.html.markdown

```diff
@@ -316,6 +323,12 @@ In addition to all arguments above, the following attributes are exported:
     - DELETED
     - REPAIRING
 
+### Plugin
+Containing key-value pairs that tag and categorize the database user. Each key and value has a maximum length of 255 characters.
```

Comment:
Contains key-value pairs that tag and categorize the cluster. Each key and value has a maximum length of 255 characters.